cmake/BuildLibCURL.cmake

```diff
@@ -1,9 +1,9 @@
 # A macro to build the bundled libcurl
-macro(curl_build)
+macro(curl_build CFLAGS)
     set(LIBCURL_SOURCE_DIR ${PROJECT_SOURCE_DIR}/third_party/curl)
     set(LIBCURL_BINARY_DIR ${PROJECT_BINARY_DIR}/build/curl/work)
     set(LIBCURL_INSTALL_DIR ${PROJECT_BINARY_DIR}/build/curl/dest)
-    set(LIBCURL_CMAKE_FLAGS "")
+    set(LIBCURL_CMAKE_FLAGS "-DCMAKE_C_FLAGS=${CFLAGS}")
```

Comment:
It seems, it is replaced by `CMAKE_C_FLAGS` on line 17 below.

Reply:
Thanks, fixed.